src/main/java/baza/trainee/constants/MailConstants.java

```diff
@@ -0,0 +1,24 @@
+package baza.trainee.constants;
+
+public final class MailConstants {
+    private MailConstants() {
+    }
+
+    public static final String MUSEUM_EMAIL = "museum@gmail.com";
```

Comment:
Should we set these parameters (MUSEUM_EMAIL, MUSEUM_LABEL, MUSEUM_TEMPLATE_PATH, USER_TEMPLATE_PATH) in application.yml?

Reply:
fixed